packages/website/components/Topbar.tsx

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import tokens from '@contentful/f36-tokens';
+import { Subheading, Flex } from '@contentful/f36-components';
 import { css } from 'emotion';
 import Link from 'next/link';
+import { DocSearch } from './DocSearch';
 
 export const TopbarHeight = '70px';
 
 const styles = {
   header: css`
-    display: flex;
-    background-color: ${tokens.blue700};
-    color: #fff;
+    display: grid;
+    grid-template-columns: 320px auto 320px;
+    column-gap: 0;
+    row-gap: 0;
+    background-color: ${tokens.colorWhite};
+    color: ${tokens.blue700};
     padding: 0 ${tokens.spacingXl};
     height: ${TopbarHeight};
-    align-items: center;
-    box-shadow: 0px 0px 10px 0px rgba(0, 0, 0, 0.2);
+    box-shadow: inset 0px -1px 0px #e7ebee;
```

Comment:
inset box-shadow?

is that correct @domarku?
In the design I have it's a border

Reply:
hmm, I will double-check that, but I think I took it from Figma. thx

Reply:
we don't have this box shadow in our tokens. I will do the border for now. we will follow up with Dom.